### PR TITLE
Blog Post for cycle_app

### PR DESCRIPTION
**Pull request by sandertyu**
I'm not sure if the header section has all the right information, please let me know if I should make some changes to the tags or category, or if I missed something while proofreading. Otherwise, I think this blog post is ready.

### CODE REVIEW

File: content/bicycle-web-app.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
+Creating a Web Application GUI for BicycleParameters
+==========================================================================
```
Comment (moorepants) [inline content/bicycle-web-app.rst:2]:
```suggestion
====================================================
```

File: content/bicycle-web-app.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
+Creating a Web Application GUI for BicycleParameters
```
Comment (moorepants) [inline content/bicycle-web-app.rst:1]: A Web Application for Analyzing Bicycle Dynamics

File: content/bicycle-web-app.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+==========================================================================
+
+:date: 2020-06-28 00:00:00
```
Comment (moorepants) [inline content/bicycle-web-app.rst:4]:
```suggestion
:date: 2020-07-03 00:00:00
```

File: content/bicycle-web-app.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+:tags: BicycleParameters, web application, gui,
```
Comment (moorepants) [inline content/bicycle-web-app.rst:5]:
```suggestion
:tags: bicycle, web application 
```

File: content/bicycle-web-app.rst [hunk 5]
```diff
@@ -0,0 +1,52 @@
+Creating a Web Application GUI for BicycleParameters
+==========================================================================
+
+:date: 2020-06-28 00:00:00
+:tags: BicycleParameters, web application, gui, 
+:category: software
+:slug: BicycleParameters-web-application
```
Comment (moorepants) [inline content/bicycle-web-app.rst:7]:
```suggestion
:slug: bicycle-dynamics-web-application
```

File: content/bicycle-web-app.rst [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+:authors: Noah Sanders
+:summary: Blog post on creating a web application to serve as a GUI for
+		  BicycleParameters
```
Comment (moorepants) [inline content/bicycle-web-app.rst:10]:
```suggestion
          bicycle dynamics.
```

File: content/bicycle-web-app.rst [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+a graphical user interface for the program as well as to enable a broader range
+of users to access it via the web. The web app takes heavy inspiration from `Jbike6 
+<http://ruina.tam.cornell.edu/research/topics/bicycle_mechanics/JBike6_web_folder/index.htm>`__,
+but aims to remove the accessibility barriers inherent to MATLAB.
+
+Building the App
+----------------
+
+Given that BicycleParameters was to be used as the backend for the website, it made sense
+to use another python based program to build the frontend. `Dash <https://dash.plotly.com/>`__
```
Comment (moorepants) [inline content/bicycle-web-app.rst:29]:
```suggestion
to use another Python based program to build the frontend. `Dash <https://dash.plotly.com/>`__
```

File: content/bicycle-web-app.rst [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+-------------
+
+The Bicycle Dynamics Web App provides a graphical interface with 10 different
+default parameter sets, a set of editable tables where users may adjust the parameter
+values as they see fit, and two plots which are generated behind the scenes
+from the datatable values. The geometry plot displays the essential schematic and centers
+of mass of the bicycle, while the eigenvalue plot reveals the self-stability of the bicycle
+as a function of its speed. Users may toggle the centers of mass and intertia ellipsoid
+on the geometry plot, and they may adjust the speed range for the eigenvalue plot.
+
```
Comment (moorepants) [inline content/bicycle-web-app.rst:46]: You need a screenshot of the app here.

**Comment by moorepants** [comment]
Made some suggestions. The only thing missing is a screenshot of the app. Take one and email it to me or post here and I'll upload to my object store for the website.

**Comment by sandertyu** [comment]
Alright, I've pushed your changes. Here's the image I would like to include. 
blog-post-app

**Comment by moorepants** [comment]
LGTM! You are done with this project :) Congrats. Thanks for all the great work.

**Comment by moorepants** [comment]
FYI: https://twitter.com/moorepants/status/1279119336225964032

**Comment by sandertyu** [comment]
Haha thank you very much, it's been a pretty good time!!

**Pull request merged**